mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -120,6 +120,12 @@
 		return __original(_change, _difficulty, _type, _showIconBeforeMoraleIcon, _noNewLine);
 	}
 
+	q.onRoundStart = @(__original) function()
+	{
+		if (this.isAlive() && ::Time.getRound() == 1) this.getSkills().add(::new("scripts/skills/special/rf_first_turn_initiative"));
```

Comment:
In early parts of the code base we didn't always new-line `if` statements, but I know Enduriel is very adamant on this that they should always be newlined. So it should be:
```squirrel
if (this.isAlive() && ::Time.getRound() == 1)
	this.getSkills().add(::new("scripts/skills/special/rf_first_turn_initiative"));
```